takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -5,68 +5,65 @@
 
 {% block dropdown_id %}nav-about{% endblock %}
 
-{% block dropdown_title %}About Takwimu{% endblock %}
-{% block dropdown_desc %}Takwimu provides quality insights and data-driven analysis to empower African development champions.{% endblock %}
+{% block dropdown_title %}<p class="text-right pr-3">About Takwimu</p>{% endblock %}
+{% block dropdown_desc %}<p class="text-right pr-3 font-weight-light">Takwimu provides quality insights and data-driven analysis to empower African development champions.</p>{% endblock %}
 
 {% block dropdown_content %}
 <div class="row pl-4 pb-5" style="width: 100%;">
   <div class="col-sm-4">
     <ul style="color:#7ab1f2; font-weight: bold;">
       <li class="pb-2" type="disc"><a href="{% url 'about_page' %}"><u>About Takwimu</u></a></li>
-      <li class="pb-2" type="disc"><a href="/about/support-services"><u>Support Services</u></a></li>
+      <li class="pb-2" type="disc"><a href="/about/support-services"><u>Our Data Services</u></a></li>
+      <li class="pb-2" type="disc"><a href="https://medium.com/takwimu-africa"><u>Insights &amp; Stories</u></a></li>
       <li class="pb-2" type="disc"><a href="/contact-us"><u>Contact Us</u></a></li>
       <li class="mb-0 pb-3" type="disc"><a href="/dashboard"><u>Dashboard (Admin)</u></a></li>
     </ul>
   </div>
-  <div class="col-sm-4">
-    <h5 class="pb-2">Follow us</h5>
+  <div class="col-sm-8">
+    <p class="font-weight-bold pb-0">Follow us</p>
     <div class="mt-2 d-inline-block p-2 rounded" style="background-color:#1455a0;">
-      <ul class="navbar-nav">
-        {% to_list 'twitter' 'facebook' 'instagram' 'slack' 'github' as social_medias %}
+
+        <ul class="navbar-nav">
+        {% to_list 'twitter' 'facebook' 'medium' 'github' as social_medias %}
         {% for social_media in social_medias %}
           {% if settings.takwimu.SocialMediaSetting|object_value:social_media %}
-            <li class="nav-item p-2">
-              <a class="nav-link{% if not forloop.last %} border-right {% else %} {% endif %}" 
-                href="{{ settings.takwimu.SocialMediaSetting|object_value:social_media }}" target="_blank" rel="noopener">
+            <li class="nav-item">
+              <a class="nav-link float-left" href="{{ settings.takwimu.SocialMediaSetting|object_value:social_media }}"
+                target="_blank" rel="noopener">
                 <i class="fab fa-{{ social_media }} fa-fw"></i>
               </a>
+              {% if not forloop.last %}<div class="border-right m-2 float-right" style="height:24px; width:1px; opacity:0.2;"></div> {% endif %}
```

Comment:
Border implementation should be as [before](https://github.com/TakwimuAfrica/TAKWIMU/pull/303/files#diff-1bce4728246b2491638e25b72168943aL29) and implement a style tag using something similar to:

```
<style>
...
#.nav-about .navbar-nav .border-right {
  STYLE_GOES_HERE
}
...
</style>
```